src/ClientApp/package.json

```diff
@@ -6,81 +6,58 @@
-    "build:development": "env-cmd --environments development dotnet publish ../parole.csproj -c Release -r win10-x64 --self-contained false && npm run copy:build",
-    "build:test": "react-scripts build",
-    "copy:arcgis": "npm run copy:public && npm run copy:build",
-    "copy:public": "ncp ./node_modules/@arcgis/core/assets ./public/assets",
-    "copy:build": "ncp ./node_modules/@arcgis/core/assets ../bin/Release/net5.0/win10-x64/publish/ClientApp/build/assets",
-    "eject": "react-scripts eject",
+    "build-storybook": "build-storybook",
+    "build": "vite build",
+    "copy:arcgis": "cpy \"./node_modules/@arcgis/core/assets/**\" ./public/assets",
+    "dev": "vite",
     "format": "prettier --write src --ignore-path ./.gitignore",
-    "lint": "eslint src",
+    "lint": "eslint ./src --ext .jsx,.js --ignore-path ./.gitignore",
+    "preview": "vite preview",
     "release": "standard-version",
-    "start:all": "env-cmd --environments development dotnet run -p ../parole.csproj",
-    "start": "env-cmd --environments development react-scripts start",
-    "storybook": "start-storybook -p 9009 -s public",
-    "test:debug": "react-scripts --inspect-brk test --runInBand --no-cache",
-    "test": "react-scripts test",
-    "wonkavise:stage:clean": "wonkavision clean ./build ./../bin ./../../deploy",
-    "wonkavise:stage:extract": "env-cmd --environments stage wonkavision unzip parole deploy.zip --extra-command \"rm app_offline.htm\"",
-    "wonkavise:stage:ship": "env-cmd --environments stage wonkavision ship ./../../deploy/deploy.zip parole && env-cmd --environments stage wonkavision ship ../app_offline.htm parole",
-    "wonkavise:stage:zip": "wonkavision zip ../bin/Release/net5.0/win10-x64/publish ./../../deploy deploy.zip",
-    "wonkavise:stage": "npm run wonkavise:stage:clean && npm run build:stage && npm run wonkavise:stage:zip && npm run wonkavise:stage:ship && npm run wonkavise:stage:extract",
-    "wonkavise:production:clean": "wonkavision clean ./build ./../bin ./../../deploy",
-    "wonkavise:production:extract": "env-cmd --environments production wonkavision unzip parole deploy.zip --extra-command \"rm app_offline.htm\"",
-    "wonkavise:production:ship": "env-cmd --environments production wonkavision ship ./../../deploy/deploy.zip parole && env-cmd --environments production wonkavision ship ../app_offline.htm parole",
-    "wonkavise:production:zip": "wonkavision zip ../bin/Release/net5.0/win10-x64/publish ./../../deploy deploy.zip",
-    "wonkavise:production": "npm run wonkavise:production:clean && npm run build:production && npm run wonkavise:production:zip && npm run wonkavise:production:ship && npm run wonkavise:production:extract"
+    "start:all": "env-cmd --environments development dotnet run --project ../parole.csproj",
+    "start": "vite",
+    "sw": "node build-service-worker.js",
```

Comment:
**question**: This is no longer relevant, no?

Reply:
yep, i will clean that up thanks.